CompilerSource/backend/GameData.cpp

```diff
@@ -305,32 +305,32 @@ ObjectData::ObjectData(const ::GmObject &object, const ESLookup &lookup):
 }
 
 RoomData::RoomData(const buffers::resources::Room &q, const std::string& name):
-  buffers::resources::Room(q), name(name) {}
+  BaseProtoWrapper(q), name(name) {}
```

Comment:
I can't understand why these initialize `BaseProtoWrapper` which is a type definition, not a member? It seems like the initialization should be for the `ProtoMessageInheritor::data` member?

Reply:
This is just how you initialize your parent type in C++. Notice that it was `buffers::resources::Room` before (also a type).

Reply:
Ohhhhh, silly me, ok, I see.